### PR TITLE
Add usage guide

### PR DESCRIPTION
**Pull request by VijayanB**
Include usage guide to give detailed information about CLI features
and how to use CLI.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/guide/usage.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
+# Using the ODFE CLI
```
Comment (jmazanec15) [inline docs/guide/usage.md:1]: In my opinion, we should just refer to ODFE CLI by its name: odfe-cli

**Comment by VijayanB** [comment]
@aetter can you review this? Thanks

**Review by aetter — APPROVED** [review]
I was not aware of the curl stuff. That's awesome. Will test.

**Review by vamshin — APPROVED** [review]
LGTM!

**Pull request merged**